testing/test-setup/src/vitest.d.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @typescript-eslint/consistent-type-definitions,@typescript-eslint/no-empty-interface */
```

Comment:
I'll be adding exceptions for these in the shared config later, BTW:
- https://github.com/code-pushup/eslint-config/issues/27